stable/traefik/values.yaml

```diff
@@ -61,3 +61,8 @@ accessLogs:
 #  labelSelector:
 rbac:
   enabled: false
+# Enable the /metrics endpoint, for now only supports prometheus
+# set to true to enable metric collection by prometheus
+metrics:
+  enabled: false
+  prometheus.buckets: 0.1,0.3,1.2,5
```

Comment:
@dhilipkumars Interesting. Last I checked only Prometheus was supported. Datadog and statsd support was added not too long ago(*) and wasn't referenced in [their docs](https://docs.traefik.io/toml/) as of yet.

Given that they do support multiple metrics provider, I agree we should allow setting provider in the chart.

Thanks for checking!

* -
 https://github.com/containous/traefik/commit/69c628b626a01e40dda33368534e1b8a59791bda